google-cloud-spanner/src/main/java/com/google/cloud/spanner/BatchClient.java

```diff
@@ -59,4 +59,13 @@ public interface BatchClient {
    *     successive read/query.
    */
   BatchReadOnlyTransaction batchReadOnlyTransaction(BatchTransactionId batchTransactionId);
+
+  /**
+   * Returns the {@link DatabaseRole} used by the client connection. The database role that is used
```

Comment:
nit: replace the `{@link DatabaseRole}` with just `database role`. `DatabaseRole` is not a valid class name, so it cannot be linked.